sleepy_discord/websocketpp_websocket.h

```diff
@@ -1,7 +1,12 @@
 #pragma once
-#define ASIO_STANDALONE
-#define _WEBSOCKETPP_CPP11_RANDOM_DEVICE_
-#define _WEBSOCKETPP_CPP11_TYPE_TRAITS_
+
+#ifdef _MSC_VER
+	#pragma message(": warning: Forcing non-boost version of asio")
```

Comment:
I don't know if this is a bug or not, but compiling without websocketpp and with uwebsockets, prints out this warning.